test/util.go

```diff
@@ -96,12 +97,34 @@ func TypeOnCanvas(c fyne.Canvas, chars string) {
 	typeChars([]rune(chars), c.OnTypedRune())
 }
 
+// WaitForThemeToBeApplied waits for the current theme to be applied to the current app.
+func WaitForThemeToBeApplied(t *testing.T) {
```

Comment:
How about making this `ApplyTheme`, take the theme parameter and do the settings apply as well.
Every call of it follows a setTheme, so why not bring it into this method :)